auto_round/data_type/nvfp.py

```diff
@@ -222,6 +224,14 @@ def fp4_v2(tensor, bits=4, group_size=32, v=0, max_scale=1.0, **kwargs):
 
 
 if __name__ == "__main__":
```

Comment:
Please help move tests into the test folder, others LGTM.

Reply:
Will be later. Currently many tests exists in code, we need to handle them in batch @wenhuach21